notebooks/community/pipelines/google_cloud_pipeline_template.ipynb

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
<p>Add heading to cell.</p><p> </p><p>Vertex Endpoint -&gt; Vertex AI Endpoint</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>